background_scripts/main.js

```diff
@@ -868,6 +951,7 @@ Object.assign(globalThis, {
   HintCoordinator,
   BackgroundCommands,
   majorVersionHasIncreased,
+  nextZoomLevel,
```

Comment:
I saw that this was how a previous coder had exposed `majorVersonHasIncreased` for testing purposes, so I did the same for `nextZoomLevel`. If you have a better preferred method, I'd love to use it.